FAILED: SemanticException [Error 10072]: Database does not exist: testdb
NoViableAltException(17@[192:1: tableName : (db= identifier DOT tab= identifier -> ^( TOK_TABNAME $db $tab) |tab= identifier -> ^( TOK_TABNAME $tab) );])
	at org.antlr.runtime.DFA.noViableAlt(DFA.java:158)
	at org.antlr.runtime.DFA.predict(DFA.java:144)
	at org.apache.hadoop.hive.ql.parse.HiveParser_FromClauseParser.tableName(HiveParser_FromClauseParser.java:4753)
	at org.apache.hadoop.hive.ql.parse.HiveParser.tableName(HiveParser.java:45667)
	at org.apache.hadoop.hive.ql.parse.HiveParser.alterStatement(HiveParser.java:7394)
	at org.apache.hadoop.hive.ql.parse.HiveParser.ddlStatement(HiveParser.java:2685)
	at org.apache.hadoop.hive.ql.parse.HiveParser.execStatement(HiveParser.java:1650)
	at org.apache.hadoop.hive.ql.parse.HiveParser.statement(HiveParser.java:1109)
	at org.apache.hadoop.hive.ql.parse.ParseDriver.parse(ParseDriver.java:202)
	at org.apache.hadoop.hive.ql.parse.ParseDriver.parse(ParseDriver.java:166)
	at org.apache.hadoop.hive.ql.Driver.compile(Driver.java:399)
	at org.apache.hadoop.hive.ql.Driver.compile(Driver.java:311)
	at org.apache.hadoop.hive.ql.Driver.compileInternal(Driver.java:1150)
	at org.apache.hadoop.hive.ql.Driver.runInternal(Driver.java:1198)
	at org.apache.hadoop.hive.ql.Driver.run(Driver.java:1087)
	at org.apache.hadoop.hive.ql.Driver.run(Driver.java:1077)
	at org.apache.hadoop.hive.cli.CliDriver.processLocalCmd(CliDriver.java:216)
	at org.apache.hadoop.hive.cli.CliDriver.processCmd(CliDriver.java:168)
	at org.apache.hadoop.hive.cli.CliDriver.processLine(CliDriver.java:386)
	at org.apache.hadoop.hive.cli.CliDriver.processLine(CliDriver.java:321)
	at org.apache.hadoop.hive.cli.CliDriver.processReader(CliDriver.java:419)
	at org.apache.hadoop.hive.cli.CliDriver.processFile(CliDriver.java:435)
	at org.apache.hadoop.hive.cli.CliDriver.executeDriver(CliDriver.java:731)
	at org.apache.hadoop.hive.cli.CliDriver.run(CliDriver.java:698)
	at org.apache.hadoop.hive.cli.CliDriver.main(CliDriver.java:634)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:232)
	at org.apache.hadoop.util.RunJar._main(RunJar.java:148)
	at org.apache.hadoop.util.RunJar.access$000(RunJar.java:52)
	at org.apache.hadoop.util.RunJar$1.run(RunJar.java:139)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1690)
	at org.apache.hadoop.security.SecurityUtil.doAsConfigUser(SecurityUtil.java:649)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
FAILED: ParseException line 2:12 cannot recognize input near '.' '.' 'SET' in table name
FAILED: SemanticException [Error 10001]: Table not found log_messages
FAILED: SemanticException [Error 10001]: Table not found log_messages
ls: `/user/hive/warehouse/testdb.db/log_messages/year=2011/month=1/day=1': No such file or directory
Command failed with exit code = 1
Query returned non-zero code: 1, cause: null
ls: `/user/hive/warehouse/testdb.db/log_messages/year=2011/month=1/day=2': No such file or directory
Command failed with exit code = 1
Query returned non-zero code: 1, cause: null
ls: `/user/hive/warehouse/testdb.db/log_messages/year=2011/month=2/day=1': No such file or directory
Command failed with exit code = 1
Query returned non-zero code: 1, cause: null
FAILED: SemanticException [Error 10001]: Table not found testdb.log_messages
FAILED: SemanticException [Error 10001]: Table not found testdb.log_messages
FAILED: SemanticException [Error 10001]: Table not found testdb.log_messages
FAILED: SemanticException [Error 10001]: Table not found testdb.log_messages
FAILED: SemanticException [Error 10001]: Table not found testdb.log_messages
FAILED: SemanticException [Error 10001]: Table not found testdb.log_messages
rm: `/user/hive/warehouse/testdb.db/log_messages/year=2011/month=1/day=1': No such file or directory
Command failed with exit code = 1
Query returned non-zero code: 1, cause: null
rm: `/user/hive/warehouse/testdb.db/log_messages/year=2011/month=1/day=2': No such file or directory
Command failed with exit code = 1
Query returned non-zero code: 1, cause: null
rm: `/user/hive/warehouse/testdb.db/log_messages/year=2011/month=2/day=1': No such file or directory
Command failed with exit code = 1
Query returned non-zero code: 1, cause: null
OK
